packages/main/test/pages/Carousel.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+
+	<title>Carousel</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); padding: 1rem;">
+<style>
+	ui5-carousel {
+		height: 20rem;
+		margin-bottom: 1rem;
+	}
+</style>
+<ui5-carousel id="carousel1" cycling="true">
+	<ui5-button>Button 1</ui5-button>
+	<ui5-button>Button 2</ui5-button>
+	<ui5-button>Button 3</ui5-button>
+</ui5-carousel>
+
+<ui5-carousel id="carousel2">
+	<ui5-button>Button 1</ui5-button>
+	<ui5-button>Button 2</ui5-button>
+	<ui5-button>Button 3</ui5-button>
+	<ui5-button>Button 4</ui5-button>
+	<ui5-button>Button 5</ui5-button>
+	<ui5-button>Button 6</ui5-button>
+	<ui5-button>Button 7</ui5-button>
+	<ui5-button>Button 8</ui5-button>
+	<ui5-button>Button 9</ui5-button>
+</ui5-carousel>
+
+<ui5-carousel id="carousel3" arrows-placement="Navigation">
+	<ui5-button>Button 1</ui5-button>
+	<ui5-button>Button 2</ui5-button>
+	<ui5-button>Button 3</ui5-button>
+</ui5-carousel>
+
+<ui5-carousel id="carousel4">
```

Comment:
Still, on mobile (iPhone 8+) the 2nd and 3rd page covers the dots. 
I saw that you  decreased the list items count and this improved the sample, but I think this only hides the issue.

I think the content should be below the dots for sure and if it`s bigger in height we should be able to scroll it up and down.